models/yang/extensions/openconfig-lldp-ext.yang

```diff
@@ -0,0 +1,25 @@
+module openconfig-lldp-ext {
+
+  yang-version "1";
+
+  // namespace
+  namespace "http://openconfig.net/yang/lldp/extention";
+
+  prefix "oc-lldp-ext";
+
+  import openconfig-lldp { prefix oc-lldp; }
+  import openconfig-lldp-types { prefix oc-lldp-types; }
+  import openconfig-interfaces { prefix oc-if; }
+  import ietf-yang-types { prefix yang; }
+  import openconfig-extensions { prefix oc-ext; }
```

Comment:
I only see `import openconfig-lldp { prefix oc-lldp; }` used are the test of the imports needed?

Reply:
Removed the other imports.